dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/KernelBootstrap.java

```diff
@@ -135,12 +142,7 @@ public ReturnCode go() {
 
             // Find the bootstrap resources we need to launch the nested framework.
             // MAY THROW if these resources can not be found or read
-            KernelResolver resolver = new KernelResolver(bootProps.getInstallRoot(),
-                            bootProps.getWorkareaFile(KernelResolver.CACHE_FILE),
-                            bootManifest.getKernelDefinition(bootProps),
-                            bootManifest.getLogProviderDefinition(bootProps),
-                            bootManifest.getOSExtensionDefinition(bootProps),
-                            libertyBoot);
+            KernelResolver resolver = new KernelResolver(bootProps.getInstallRoot(), bootProps.getWorkareaFile(KernelResolver.CACHE_FILE), bootDefaults.getKernelDefinition(bootProps), bootDefaults.getLogProviderDefinition(bootProps), bootDefaults.getOSExtensionDefinition(bootProps), libertyBoot);
```

Comment:
Not sure what happened to the formatting here.  Perhaps just a glitch in the web UI for the PR.